specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-06-01/containerregistry_build.json

```diff
@@ -0,0 +1,2578 @@
+      "get": {
+        "tags": [
+          "Tasks"
+        ],
+        "description": "Get the properties of a specified task.",
+        "operationId": "Tasks_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/TaskNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/Task"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If the registry doesn't exist, 404 (Not found) is returned."
+          }
+        },
+        "x-ms-examples": {
+          "Tasks_Get": {
+            "$ref": "./examples/TasksGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Tasks"
+        ],
+        "description": "Creates a task for a container registry with the specified parameters.",
+        "operationId": "Tasks_Create",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/TaskNameParameter"
+          },
+          {
+            "name": "taskCreateParameters",
+            "in": "body",
+            "description": "The parameters for creating a task.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Task"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/Task"
+            }
+          },
+          "201": {
+            "description": "The request was successfully accepted; the operation will complete asynchronously. The provisioning state of the resource should indicate the current state of the resource.",
+            "schema": {
+              "$ref": "#/definitions/Task"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If the registry doesn't exist, 404 (Not found) is returned. If any of the input parameters is wrong, 400(Bad Request) is returned."
```

Comment:
```suggestion
            "description": "Error response describing why the operation failed. If the registry doesn't exist, 404 (Not found) is returned. If any of the input parameters is wrong, 400 (Bad Request) is returned."
```

Reply:
@jaysterp  seems like such changes will also update all the previous swaggers, has this not been caught before - or was ignored bc of the regressive swagger updates?

Reply:
@kpajdzik can these changes be ignored? Do I need to update all the previous swaggers for this?
cc: @jaysterp

Reply:
It's a minor issue that's going to impact documentation only so it can be ignored. You don't have to fix previous swaggers.